### PR TITLE
SCRUM-7_Get_Task

### PR DESCRIPTION
**Pull request by FreddyFZ17**
Added assert_schema_task schema validation to ensure the integrity of data obtained from the getTask endpoint. Additionally, the task.json file was introduced that defines the schema based on the current response from the endpoint. Finally, the following test cases were automated in test_GetTask.py: test_get_task_success, test_get_task_success_filter, test_get_task_list, test_get_task_unauthorized, and test_get_task_invalid_auth. These test cases verify correct task recovery, task filtering, task listing, handling of incorrect authorization and invalid authentication, respectively. This approach improves the robustness of the tests and ensures that the API meets the defined criteria.

### CODE REVIEW

File: src/assertions/assertion_schemas.py
Comment (Jasonn5) [inline src/assertions/assertion_schemas.py:1]: looks good

File: src/resources/schemas/task.json
Comment (Jasonn5) [inline src/resources/schemas/task.json:1]: approve

File: tests/task/test_GetTask.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+from src.assertions.assertion_schemas import assert_schema_task
+import requests
+from config import  BASE_URI
+from src.assertions.assertion_status import assert_status_code_ok, assert_status_code_unauthorized
+
+def test_get_task_success(get_headers):
+    url = f"{BASE_URI}/Kanban/Task?select=dateEnd%2CdateEndDate%2CparentId%2CparentType%2CparentName%2Cpriority%2Cname%2Cstatus&maxSize=5&offset=0&orderBy=createdAt&order=desc"
```
Comment (Jasonn5) [inline tests/task/test_GetTask.py]: Use Enums for params

**Review by Jasonn5 — CHANGES_REQUESTED** [review]

**Comment by FreddyFZ17** [comment]
The new testcases were added and the structure agreed upon in the team was used, in addition to trying to use good practices, new asserts and so on were added.

File: config.py
Comment (Jasonn5) [inline config.py:1]: looks good

**Review by Jasonn5 — DISMISSED** [review]

File: src/espocrm_api/calls_endpoints.py
Comment (Jasonn5) [inline src/espocrm_api/calls_endpoints.py:1]: unnecessary change

**Review by Jasonn5 — APPROVED** [review]

File: src/espocrm_api/calls_endpoints.py
Comment (Diego-Claros17) [inline src/espocrm_api/calls_endpoints.py:1]: Unnecesary change but no problem

File: config.py
Comment (Diego-Claros17) [inline config.py:1]: Well done

File: src/assertions/assertion_schemas.py
Comment (Diego-Claros17) [inline src/assertions/assertion_schemas.py:1]: Works well

File: src/resources/schemas/task.json
Comment (Diego-Claros17) [inline src/resources/schemas/task.json:1]: GJ

File: tests/task/test_GetTask.py
Comment (Diego-Claros17) [inline tests/task/test_GetTask.py:1]: All pytest marks were added. Well done Freddy bb

**Review by Diego-Claros17 — APPROVED** [review]
All code works well and does not cause any conflicts. No observations for me, GJ Freddy.
